ui/src/app/views/admin-user-bypass/component.tsx

```diff
@@ -62,7 +62,7 @@ export class AdminUserBypass extends React.Component<
   }
 
   hasEdited(): boolean {
-    return this.state.selectedModules !== this.state.initialModules;
+    return fp.xor(this.state.selectedModules, this.state.initialModules).length !== 0;
```

Comment:
This is pretty convoluted, but I don't have a better suggestion after looking through lodash docs for a bit. An alternative is `fp.isEqual(fp.sortBy(x), fp.sortBy(y))`, which is not much better.

Reply:
Yeah... I actually like the xor logic, because we are fundamentally looking for exclusive or information here. But it is kinda on the convoluted side.